lua/lspconfig.lua

```diff
@@ -76,4 +76,14 @@ function mt:__index(k)
   return configs[k]
 end
 
+local minimum_neovim_version = '0.10'
+if vim.fn.has('nvim-' .. minimum_neovim_version) == 0 then
+  local msg = string.format(
+    'nvim-lspconfig requires Nvim version %s, but you are running: %s',
+    vim.version.parse(minimum_neovim_version),
+    vim.version()
+  )
+  vim.notify(msg, vim.log.levels.WARN)
```

Comment:
since we are using vim.uv, should it just be an assert?

Reply:
I'm confused. What does this have to do with `vim.uv` (other than that the PR for it happened to inspire this one)? I imagine this to be a general-purpose warning that we can continually use whenever we need to bump the minimum version, rather than a one-time thing.

Reply:
since https://github.com/neovim/nvim-lspconfig/commit/c25b9dd6bddb125b538f3f01cb47b4a6e606f6c5 , there will be an error for any Nvim version that doesn't have vim.uv

so showing a warning here means that code will still run and produce an error

Reply:
Oh derp, you're right. I understand what you mean. Yes, aborting/throwing an error sounds like the safest bet in that case.

Reply:
Still not entirely sure if `vim.uv` is relevant, it just happens to be the last thing that broke (but in the future there might be some other function), so don't really see the value in hardcoding it to the existence of `vim.uv`. Let's throw an error instead of warning? I'll update the code and we can take a look.

Reply:
What the hell, the output from `vim.notify` is awful when used with `vim.log.levels.ERROR`. We can't use this lol.

Reply:
> so don't really see the value in hardcoding it to the existence of `vim.uv`

it's just an example. don't need to hardcode anything ,just change vim.notify() to assert()

Reply:
Sure, done